assets/css/src/_media.css

```diff
@@ -102,7 +102,7 @@ object {
 
 .gallery-item {
 	padding: 0.5em;
-	border: 1px solid rgb(222, 222, 222);
+	border: 1px solid #dedede;
```

Comment:
This should be the `#ccc` color as well, it's so similar, and also a border. Related, my other comment below.